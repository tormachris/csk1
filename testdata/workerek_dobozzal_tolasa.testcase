VerboseMode
XMLPrepare
<!DOCTYPE map
[
<!ELEMENT map>
<!ELEMENT tile(id, type, connectsto)>
<!ELEMENT id (1)
<!ELEMENT type (0)
<!ELEMENT connectsto (2)
<!ELEMENT tile(id, type, connectsto)>
<!ELEMENT id (2)
<!ELEMENT type (0)
<!ELEMENT connectsto (1)
<!ELEMENT tile(id, type, connectsto)>
<!ELEMENT id (3)
<!ELEMENT type (0)
<!ELEMENT connectsto (2)
<!ELEMENT tile(id, type, connectsto)>
<!ELEMENT id (4)
<!ELEMENT type (0)
<!ELEMENT connectsto (3)
<!ELEMENT thing(type, tileid, force)>
<!ELEMENT type (worker)
<!ELEMENT tileid (1)
<!ELEMENT force (4)
<!ELEMENT thing(type, tileid, force)>
<!ELEMENT type (crate)
<!ELEMENT tileid (2)
<!ELEMENT force (0)
<!ELEMENT thing(type, tileid, force)>
<!ELEMENT type (worker)
<!ELEMENT tileid (3)
<!ELEMENT force (0)
]>
XMLOver
moveWorker 0 NORTH
