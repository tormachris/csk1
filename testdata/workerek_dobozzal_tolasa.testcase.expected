Crate 1 : moved
Worker 2 : died
true
